src/classes/CW_AdminController.cls

```diff
@@ -0,0 +1,168 @@
+public with sharing class CW_AdminController {
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.